# Autonomous system constatns
vehicle:
  max_throttle: 0.4 # m/s^2
  target_speed: 8 # m/s
  max_steering: 0.3

cones_range_cutoff: 12.0 # max distance to cone

# Vision params
vision:
  cone_too_small_thresh: 240
  model_path: 
  conf_thres: 0.7
  iou_thres: 0.5
  cone_clf_path: /cone_clf.onnx
